pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -1,13 +1,21 @@
-{ stdenv, fetchurl, pkgconfig, dbus, glib, alsaLib,
-  python3, readline, udev, libical, systemd, fetchpatch,
-  enableWiimote ? false, enableMidi ? false, enableSixaxis ? false }:
+{ stdenv, lib, fetchurl, pkgconfig, dbus, glib, alsaLib,
+  python3, readline, udev, libical, systemd, json_c,
+  enableHealth ? false,
+  enableMesh ? false,
+  enableMidi ? false,
+  enableNfc ? false,
+  enableSap ? false,
+  enableSixaxis ? false,
+  enableWiimote ? false,
+}:
 
 stdenv.mkDerivation rec {
-  name = "bluez-5.50";
+  version = "5.51";
+  name = "bluez-${version}";
 
   src = fetchurl {
     url = "mirror://kernel/linux/bluetooth/${name}.tar.xz";
-    sha256 = "048r91vx9gs5nwwbah2s0xig04nwk14c5s0vb7qmaqdvighsmz2z";
+    sha256 = "1fpbsl9kkfq6mn6n0dg4h0il4c7fzhwhn79gh907k5b2kwszpvgb";
   };
 
   pythonPath = with python3.pkgs; [
```

Comment:
pygobject2 should not be needed since 2014 (https://github.com/RadiusNetworks/bluez/commit/7adf8d09b4ba060ecb223e36fd6678aa52ab0af1)